TP-Sampler/index.html

```diff
@@ -75,17 +74,162 @@
         <div id="measures-instructions" data-text="DG.plugin.Sampler.measures-instructions"></div>
         <label for="select-measure" id="select-measure-label" data-text="DG.plugin.Sampler.select-measure"></label>
         <select id="select-measure">
-          <option id="count" data-text="DG.plugin.Sampler.count"></option>
-          <option id="conditional-count" data-text="DG.plugin.Sampler.conditional-count"></option>
-          <option id="conditional-percentage" data-text="DG.plugin.Sampler.conditional-percentage"></option>
-          <option id="mean" data-text="DG.plugin.Sampler.mean"></option>
-          <option id="conditional-mean" data-text="DG.plugin.Sampler.conditional-mean"></option>
-          <option id="diff-of-means" data-text="DG.plugin.Sampler.diff-of-means"></option>
-          <option id="sum" data-text="DG.plugin.Sampler.sum"></option>
+          <option disabled selected value="">Select a formula</option>
+          <option id="count" value="count" data-text="DG.plugin.Sampler.count"></option>
+          <option id="sum" value="sum" data-text="DG.plugin.Sampler.sum"></option>
+          <option id="mean" value="mean" data-text="DG.plugin.Sampler.mean"></option>
+          <option id="median" value="median" data-text="DG.plugin.Sampler.median"></option>
+          <option id="conditional_count" value="conditional_count" data-text="DG.plugin.Sampler.conditional_count"></option>
+          <option id="conditional_sum" value="conditional_sum" data-text="DG.plugin.Sampler.conditional_sum"></option>
+          <option id="conditional_percentage" value="conditional_percentage" data-text="DG.plugin.Sampler.conditional_percentage"></option>
+          <option id="conditional_mean" value="conditional_mean" data-text="DG.plugin.Sampler.conditional_mean"></option>
+          <option id="conditional_median" value="conditional_median" data-text="DG.plugin.Sampler.conditional_median"></option>
+          <option id="difference_of_means" value="difference_of_means" data-text="DG.plugin.Sampler.difference_of_means"></option>
+          <option id="difference_of_medians" value="difference_of_medians" data-text="DG.plugin.Sampler.difference_of_medians"></option>
         </select>
+        <div id="measure-name-container">
+          <label for="measure-name" id="measure-name-label" data-text="DG.plugin.Sampler.measure-name-label"></label>
+          <input type="text" id="measure-name"></input>
+        </div>
+        <div id="measure-formulas">
+          <div id="count-formula-container" class="formula hidden">
+            <span id="count-formula" data-text="DG.plugin.Sampler.count-formula"></span>
+          </div>
+          <div id="sum-formula-container" class="formula hidden">
+            <span id="sum-formula" data-text="DG.plugin.Sampler.sum-formula"></span>
+            <span id="formula-paren" >(
```

Comment:
Just for consistency, remove space before `>`
```suggestion
            <span id="formula-paren">(
```